### PR TITLE
feat(popover): remove max-width

### PR DESCRIPTION
**Pull request by asangma**
re #636 

* Removed max-width
* Updated CHANGELOG
* Tweak to demo-app

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+- `calcite-popover` - `max-width` has been removed. Content may need width set.
```
Comment (macandcheese) [inline CHANGELOG.md:10]: I'd remove this second item, and then move the one under breaking change to a new `Updated` heading.

Comment (asangma) [reply CHANGELOG.md:10]: Cool.

**Comment by asangma** [comment]
> Looks good, does Popper prevent content from overflowing horizontally?

It looks like it just gets cray wide.
image

File: CHANGELOG.md (hunk 1 above)
Comment (asangma) [reply CHANGELOG.md]: Done.

**Comment by asangma** [comment]
Thanks for the help @macandcheese. Murgling!

**Pull request merged**